lib/cext/include/ruby/internal/newobj.h

```diff
@@ -57,9 +57,12 @@
  */
 #define RB_NEWOBJ_OF(obj,type,klass,flags) type *(obj) = RBIMPL_CAST((type *)rb_newobj_of(klass, flags))
 
-#define NEWOBJ     RB_NEWOBJ      /**< @old{RB_NEWOBJ} */
+// We can't support either `NEWOBJ` or `OBJSETUP` in TruffleRuby. They rely on being able to create an object
+// without an associated class and updating state that is tied directly to MRI's object layout.
+// #define NEWOBJ     RB_NEWOBJ      /**< @old{RB_NEWOBJ} */
+// #define OBJSETUP   rb_obj_setup   /**< @old{rb_obj_setup} */
```

Comment:
So in headers we import from CRuby we never change lines but only add lines (which is better for conflicts and clarity and seeing the differences), so this should be a `#ifndef TRUFFLERUBY` around these.

Reply:
Fair enough. I just wanted the comment grouped with the lines changed, but I can leave it floating above.